partner_contact_nutrition_goal/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    nutrition_goal = fields.Selection([('lose', 'Lose weight'),
```

Comment:
I feel like this would be better described as `weight_goal`. Nutritional goals is a bit too broad IMO and could also be things like specific diets, etc.